pkg/sqlite/scene.go

```diff
@@ -1234,6 +1234,23 @@ func (qb *SceneStore) SaveActivity(ctx context.Context, id int, resumeTime *floa
 	return true, nil
 }
 
+func (qb *SceneStore) ResetActivity(ctx context.Context, id int) (bool, error) {
+	if err := qb.tableMgr.checkIDExists(ctx, id); err != nil {
+		return false, err
+	}
+
+	record := goqu.Record{}
+
+	record["resume_time"] = 0.0
+	record["play_duration"] = 0.0
+
+	if err := qb.tableMgr.updateByID(ctx, id, record); err != nil {
+		return false, err
+	}
```

Comment:
This can be simplified to:
```
if err := qb.tableMgr.updateByID(ctx, id, goqu.Record{
	"resume_time":   0.0,
	"play_duration": 0.0,
}); err != nil {
	return false, err
}```

Or alternatively it can be in the `updateByID` call.

Reply:
After moving to boolean flags for resume/duration, the simplification is no longer possible.